core-tests/shared/src/test/scala/zio/ZIOAppSpec.scala

```diff
@@ -47,7 +47,7 @@ object ZIOAppSpec extends ZIOBaseSpec {
         }
       }
 
-      val app1 = ZIOApp(ZIO.fail("Uh oh!"), RuntimeConfigAspect.addLogger(logger1))
+      val app1 = ZIOAppDefault(ZIO.fail("Uh oh!"), RuntimeConfigAspect.addLogger(logger1))
```

Comment:
Maybe the default could be `ZIOApp` and the customizable something longer like `CustomZIOApp`?

Reply:
Yeah I was wondering about that too. I think for most "real" cases you are going to want to be able to provide some layers but it is definitely nice to have something simple for demos and prototyping.

Reply:
May also be some value in being consistent with `ZIOSpec` / `ZIOSpecDefault`, though we could change that too.